src/App.tsx

```diff
@@ -27,29 +42,44 @@ const App: React.FC = () => {
     console.log(query);
   }
 
+  const handleSortChange = (newSort: string) => {
+    setSelectedSort(newSort);
+    console.log(`Sorting movies by: ${newSort}`);
```

Comment:
**nitpick:** `console.log` is redundant here

Reply:
Removed